js/components/user-bar.js

```diff
@@ -40,6 +40,12 @@ define([
 			this.sortedJobListing = ko.computed(() => lodash.sortBy(this.jobListing(), el => -1 * el.executionId));
 			this.lastViewedTime=null;
 			this.permissionCheckWarningShown = false;
+			this.updateJobs = ko.computed(() => {
```

Comment:
`updateJobs` is never used. Probably you want to use `state.jobListing().subscribe`, don't you?
And while having `jobListing` the `newJobState` is not necessary (Furthermore newJobState works only for cohort definitions, but other types of analyses remains issue).